backend/plan/views.py

```diff
@@ -77,6 +77,92 @@
         },
     )
 )
+
+# PcxAutoSchema for the advanced registration view.
+@api_view(["POST"])
+@schema(
+    PcxAutoSchema(
+        response_codes={
+            reverse_func("advanced-registration-schedule"): {
+                "POST": {
+                    200: "[DESCRIBE_RESPONSE_SCHEMA]Response returned successfully.",
+                    201: "[UNDOCUMENTED]",
+                    400: "Invalid advanced_reg_codes, advanced_reg_sections, schedule_user, or advanced_reg_schedule_name (see response).",
+                }
+            }
+        },
+        override_request_schema={
+            reverse_func("advanced-registration-schedule"): {
+                "POST": {
+                    "type": "object",
+                    "properties": {
+                        "advanced_reg_codes": {
+                            "type": "array",
+                            "description": (
+                                "An array of courses the user is currently planning to "
+                                "take, each specified by its string full code, of the form "
+                                "e.g. CIS-120-000."
+                            ),
+                            "items": {"type": "string"},
+                        },
+                        "advanced_reg_schedule_name": {
+                            "type": "string",
+                            "description": (
+                                "Name of the advanced registration schedule as "
+                                "Specified by the user "
+                            ),
+                        },
+                    },
+                }
+            }
+        },
+        override_response_schema={
+            reverse_func("advanced-registration-schedule"): {
+                "POST": {
+                    200: {"type": "object"}
+                }
+            }
+        },
+    )
+)
+
```

Comment:
I would delete this space. Have you tried running the documentation locally to see how the auto docs look?